packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -64,6 +64,8 @@ In CMSDS v2, it will be changed to
 
 Imports to `@cmsgov/design-system-layout` or `@cmsgov/design-system-support` are now included in the main `scss` entry point. If you would like to only import specific styles, see the `@cmsgov/design-system` [`README.md`](https://github.com/CMSgov/design-system/blob/master/packages/design-system/README.md#file-structure) for more info on our SASS architecture.
```

Comment:
Could we add a section on focus styles? Maybe explaining how it was previously released in v3 but it was moved under a flag because it is still under beta development. I feel like focus styles are pretty big so it makes sense to add some sort of explanation

Reply:
Yeah, I'll add some info explaining 👍